core/primitives/src/test_utils.rs

```diff
@@ -36,17 +36,27 @@ impl Transaction {
         receiver_id: AccountId,
         nonce: Nonce,
         block_hash: CryptoHash,
-        priority_fee: u64,
+        priority_fee: Option<u64>,
```

Comment:
nit: I think it would be cleaner to have separate methods for creating v0 and v1. It would be more explicit.

Reply:
oh wait actually theres one more call when compiling with test_features, so adding another `new_v1()` to call it there